sdk/trace/trace_test.go

```diff
@@ -217,13 +217,26 @@ func TestSetName(t *testing.T) {
 	}
 }
 
-func TestRecordingIsOn(t *testing.T) {
-	tp := NewTracerProvider()
-	_, span := tp.Tracer("Recording on").Start(context.Background(), "StartSpan")
-	defer span.End()
-	if span.IsRecording() == false {
-		t.Error("new span is not recording events")
-	}
+func TestSpanIsRecording(t *testing.T) {
+	t.Run("is recording on", func(t *testing.T) {
+		tp := NewTracerProvider()
+		_, span := tp.Tracer("Recording on").Start(context.Background(), "StartSpan")
+		defer span.End()
+		if span.IsRecording() == false {
+			t.Error("new span is not recording events")
+		}
```

Comment:
```suggestion
		assert.True(t, span.IsRecording(), "The span should be recording events")
```

This file uses both `t.Error()` and the `assert` package, but I think generally we've moved to using `assert` in most tests.